content/zh-cn/docs/tasks/debug/debug-application/debug-service.md

```diff
@@ -314,7 +318,6 @@ traffic to `hostnames-*` Pods, these need to be reviewed.
 
 Please refer to [Network Policies](/docs/concepts/services-networking/network-policies/) for more details.
 -->
-
 ## 是否存在影响目标 Pod 的网络策略入站规则？
```

Comment:
```suggestion
## 是否存在影响目标 Pod 的网络策略入站规则？   {#any-network-policy-ingress-rules-affecting-the-target-pods}
```

Reply:
Okay, now I know it's unnecessary to add not-in-use link anchors.

Reply:
how do I know if these anchors in-use or not?

Reply:
You can make a good guess when you manually input a lengthy, verbose anchor ID. If you are referencing a target using things like `#any-network-policy-ingress-rules-affecting-the-target-pods`, you know something is wrong.

Reply:
interesting, got it 👍